src/Svelte/src/SvelteBundle.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Svelte;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+/**
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ * @author Thomas Choquet <thomas.choquet.pro@gmail.com>
+ *
+ * @final
+ *
+ * @experimental
```

Comment:
You can remove the `@experimental`. I think we'll release this NOT as experimental.